client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfillOperator.kt

```diff
@@ -102,8 +104,8 @@ internal class BackfillOperator<E : DbEntity<E>, Pkey : Any> internal constructo
       val minmax = queryFactory.dynamicQuery(backfill.entityClass)
           .dynamicUniqueResult(session) { criteriaBuilder, queryRoot ->
             criteriaBuilder.tuple(
-                criteriaBuilder.min(queryRoot.get(backfill.primaryKeyHibernateName())),
-                criteriaBuilder.max(queryRoot.get(backfill.primaryKeyHibernateName())))
+                criteriaBuilder.min(backfill.getPrimaryKeyPath(queryRoot)),
```

Comment:
@swankjesse My only objection is that I can see these kinds of situations being somewhat common for performance reasons. Although maybe we just need a better way of specifying projections?